engines/sci/graphics/controls32.cpp

```diff
@@ -41,7 +41,20 @@ GfxControls32::GfxControls32(SegManager *segMan, GfxCache *cache, GfxText32 *tex
 	_gfxCache(cache),
 	_gfxText32(text),
 	_overwriteMode(false),
-	_nextCursorFlashTick(0) {}
+	_nextCursorFlashTick(0)
+{
+	_lastScrollWindowId = make_reg(0, 9999);
```

Comment:
Is this number from SSCI, like the Plane/ScreenItem `_nextObjectId`, or a thing we invented?